sce.py

```diff
@@ -37,6 +48,8 @@
         test_project.handle_testing()
     elif args.command == 'run':
         handler = SceServiceHandler(args.service)
+        if args.dbpath:
+            set_mongo_path(handler, args.dbpath)
```

Comment:
instead of doing this we can add the dbpath to the constructor like
- [ ] allow SceServiceHandler to have a new db path variable (defaults to `None`)
```py
class SceServiceHandler:
  # ...
  def __init__(self, services, dbpath=None):
    if dbpath:
      self.mongo_volume_path = dbpath
    else:
      # use the default path
       # may need to implement some logic to verify
      self.mongo_volume_path = "whatever it is"
    pass
```

- [ ] call the constructor with a new parameter
```py
handler = SceServiceHandler(args.service, args.dbpath)
```